octobot_trading/personal_data/orders/types/market/sell_market_order.py

```diff
@@ -18,5 +18,5 @@
 
 
 class SellMarketOrder(market_order.MarketOrder):
-    def __init__(self, trader):
+    def __init__(self, trader, side=None):
```

Comment:
?

Reply:
buy/sellmarket orders are not respecting the parent class constructor. It creates a need to a special case every time to check if the side should be provided or not, I think it's better to allow the parameter and not use it